pyCGM_Single/pycgmKinetics.py

```diff
@@ -19,40 +19,294 @@
 #helper functions useful for dealing with frames of data, i.e. 1d arrays of (x,y,z)
 #coordinate. Also in Utilities but need to clean everything up somewhat!         
 def f(p, x):
+    """
+    Parameters
+    ----------
+    p : list
+        A that has a length of at least 2.
+    x : int or float
+        Scaling factor for the first variable in p.
+    Returns
+    -------
+    int or float
+        Returns the first value in p scaled by x, aded by the second value in p.
+
+    Example
+    -------
+    >>> import numpy as np
+    >>> from .pycgmKinetics import f
+    >>> p = [1, 2]
+    >>> x = 10
+    >>> f(p, x) #doctest: +NORMALIZE_WHITESPACE
+    12
+
+    >>> p = np.array([5.16312215, 8.79307163])
+    >>> x = 2.0
+    >>> np.around(f(p, x),8) #doctest: +NORMALIZE_WHITESPACE
+    19.11931593
+    """
     return (p[0] * x) + p[1]
 
 def dot(v,w):
+    """Calculate the Dot Product function.
+    
+    Parameters
+    ----------
+    v : list
+        First 3-element list.
+    w : list
+        Second 3-element list.
+
+    Returns
+    -------
+    int or float
+        The quotient of the dot product of vectors v and w.
+
+    Example
+    -------
+    >>> import numpy as np
+    >>> from .pycgmKinetics import dot
+    >>> v = [1, 2, 3]
+    >>> w = [4, 5, 6]
+    >>> dot(v,w) #doctest: +NORMALIZE_WHITESPACE
```

Comment:
You shouldn't need normalize_whitespace for these tests. It is just adding clutter.  It really should almost never be needed.  There are a few times I used it when some external library was making it difficult to use on nested arrays.

Reply:
Understood, I will remove the normalize_whitespace doctests where the answer is one line, and/or works well without it.